stores/moderationSubmissionsStore.ts

```diff
@@ -101,6 +101,21 @@ export const useModerationSubmissionsStore = defineStore(
             }
         }
 
+        async function approveSubmission() {
+            if (!selectedSubmissionId.value) {
+                console.error('Submission Id is required')
```

Comment:
This is hardcoded because using translatable strings within stores in Pinia from our app causes the build to fail as it is expected in the setup of a Vue component

Reply:
Hmmm sounds like it's running during compilation time which it shouldn't be 🤔. It's fine for now but we should look into this for pinia stores running at the wrong time